dev/Scroller/Scroller.cpp

```diff
@@ -7503,13 +7553,40 @@ void Scroller::HookContentPropertyChanged(
 {
     if (content)
     {
-        if (!m_contentHorizontalAlignmentChangedToken)
+        if (auto contentAsFE = content.try_as<winrt::FrameworkElement>())
         {
-            m_contentHorizontalAlignmentChangedToken = RegisterPropertyChanged(content, winrt::FrameworkElement::HorizontalAlignmentProperty(), { this, &Scroller::OnContentPropertyChanged });
-        }
-        if (!m_contentVerticalAlignmentChangedToken)
-        {
-            m_contentVerticalAlignmentChangedToken = RegisterPropertyChanged(content, winrt::FrameworkElement::VerticalAlignmentProperty(), { this, &Scroller::OnContentPropertyChanged });
+            if (!m_contentMinWidthChangedRevoker)
+            {
+                m_contentMinWidthChangedRevoker = RegisterPropertyChanged(contentAsFE, winrt::FrameworkElement::WidthProperty(), { this, &Scroller::OnContentPropertyChanged });
```

Comment:
MinWidth ? Looks like you are registring to the same properties below. copy paste bug ?

Reply:
Oops, yes - last minute copy/paste errors. Thanks.